doc/api/http2.md

```diff
@@ -342,6 +342,31 @@ acknowledgement for a sent SETTINGS frame. Will be `true` after calling the
 `http2session.settings()` method. Will be `false` once all sent SETTINGS
 frames have been acknowledged.
 
+#### http2session.ping(callback)
```

Comment:
Could ping expose an (optional) 8-octet `payload` param?

Reply:
Yeah, I'm considering that, just not sure how useful it's going to be if we stick with the one at a time limit.

Reply:
Some considerations:
- Analysis/performance tools might want to send PING floods.
- When RTT exceeds keepalive interval. (Ouch! 😵)
- Avoid global state. Or at least do not expose it to the user.
- Follow what the spec says. Implementation is no place for protocol design.